changelog.d/46312.misc

```diff
@@ -0,0 +1 @@
+Use the same notification rules than web / iOs client
```

Comment:
is it worth calling out that the change will mean we will notify for all (new) messages?

Reply:
yes we can, i update the message